javav2/example_code/ssm/src/main/java/com/example/scenario/SSMScenario.java

```diff
@@ -0,0 +1,558 @@
+                instanceId - The ID of the instance (default: i-0149338494ed95f06).
+                title - The title of the parameter (ie, Disk Space Alert).
+                source - The source of the parameter (default: EC2).
+                category - The category of the parameter (default: Performance).
+                severity - The severity of the parameter (default: 2).
+        """;
+
+        if (args.length != 5) {
+            System.out.println(usage);
+            System.exit(1);
+        }
+
+        Scanner scanner = new Scanner(System.in);
+        String documentName;
+        String windowName;
+        String instanceId = args[0];
+        String title = args[1];
+        String source = args[2];
+        String category = args[3];
+        String severity = args[4];
+
+        Region region = Region.US_EAST_1;
+        SsmClient ssmClient = SsmClient.builder()
+            .region(region)
+            .build();
+
+        System.out.println(DASHES);
+        System.out.println("""
+            Welcome to the AWS Systems Manager SDK Getting Started scenario.
+            This Java program demonstrates how to interact with Systems Manager using the AWS SDK for Java (v2).
+            Systems Manager is the operations hub for your AWS applications and resources and a secure end-to-end management solution.
+            The program's primary functionalities include creating a maintenance window, creating a document, sending a command to a document,
+            listing documents, listing commands, creating an OpsItem, modifying an OpsItem, and deleting Systems Manager resources.
+            Upon completion of the program, all AWS resources are cleaned up.
+            Let's get started...
+            Please hit Enter
+            """);
+        scanner.nextLine();
+        System.out.println(DASHES);
+
+        System.out.println("Create a Systems Manager maintenance window.");
+        System.out.println("Please enter the maintenance window name (default is ssm-maintenance-window):");
+        String win = scanner.nextLine();
+        windowName = win.isEmpty() ? "ssm-maintenance-window" : win;
+        String winId = createMaintenanceWindow(ssmClient, windowName);
+        System.out.println(DASHES);
+
+        System.out.println("Modify the maintenance window by changing the schedule");
+        System.out.println("Please hit Enter");
+        scanner.nextLine();
+        updateSSMMaintenanceWindow(ssmClient, winId, windowName);
+        System.out.println(DASHES);
+
+        System.out.println("Create a document that defines the actions that Systems Manager performs on your managed nodes.");
+        System.out.println("Please enter the document name (default is ssmdocument):");
+        String doc = scanner.nextLine();
+        documentName = doc.isEmpty() ? "ssmdocument" : doc;
+        createSSMDoc(ssmClient, documentName);
+
+        System.out.println("Now we are going to run a command on an EC2 instance");
```

Comment:
Can you describe the command in the document? It is confusing to me at first what command is being sent. In this case "echo Hello World".